.github/workflows/container-tests.yml

```
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        container-to-test: ["postgresql-container", "s2i-nodejs-container", "s2i-python-container"]
+        container-to-test: ["postgresql-container", "nginx", "s2i-perl-container", "s2i-python-container"]
```

```suggestion
        container-to-test: ["postgresql-container", "nginx-container", "s2i-perl-container", "s2i-python-container"]
```
Missing container suffix?

---

Looks like I am a few seconds late :).

---

Stupid error ;)